VideoLocker/src/main/java/org/edx/mobile/view/EditUserProfileFragment.java

```diff
@@ -412,24 +411,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 final Uri imageUri = CropImageActivity.getImageUriFromResult(data);
                 final Rect cropRect = CropImageActivity.getCropRectFromResult(data);
                 if (null != imageUri && null != cropRect) {
-                    executePhotoTask(new SetAccountImageTask(getActivity(), username, imageUri, cropRect) {
-                        @Override
-                        protected void onSuccess(Void aVoid) throws Exception {
-                            hideLoading();
-                        }
-
-                        @Override
-                        protected void onException(Exception e) throws RuntimeException {
-                            super.onException(e);
-                            hideLoading();
-                        }
-
-                        private void hideLoading() {
-                            if (null != viewHolder) {
-                                viewHolder.profileImageProgress.setVisibility(View.GONE);
-                            }
-                        }
-                    });
+                    final Task task = new SetAccountImageTask(getActivity(), username, imageUri, cropRect);
```

Comment:
`SetAccountImageTask` can be converted into a `Call`, consequently, making the function `executePhotoTask ` accept a `Call`type object.